src/renderers/WebGLRenderer.js

```diff
@@ -1280,7 +1280,8 @@ function WebGLRenderer( parameters = {} ) {
 				minFilter: LinearMipmapLinearFilter,
 				magFilter: NearestFilter,
 				wrapS: ClampToEdgeWrapping,
-				wrapT: ClampToEdgeWrapping
+				wrapT: ClampToEdgeWrapping,
+				useMultisampleRenderToTexture: extensions.has( 'EXT_multisampled_render_to_texture' )
```

Comment:
It seems it's not necessary to store the extensions support in each texture. It should be a global thing.

Have you considered to enhance `WebGLCapabilities` with a new property so you can do this instead:
```js
if ( capabilities.multisampledRTT === true )
```